src/rust/ide/src/double_representation/graph.rs

```diff
@@ -6,10 +6,39 @@ use crate::double_representation::definition;
 use crate::double_representation::definition::DefinitionInfo;
 use crate::double_representation::definition::DefinitionName;
 use crate::double_representation::definition::DefinitionProvider;
-use crate::double_representation::node;
+use crate::double_representation::node::NodeInfo;
 
 use ast::Ast;
 use ast::known;
+use utils::fail::FallibleResult;
+
+
+
+// =============
+// === Error ===
+// =============
+
+#[derive(Fail,Display,Debug)]
+struct IDNotFound {id:ast::ID}
```

Comment:
Please change `IDNotFound` to `IdNotFound`. I know that `ast::ID` has bad name, but it is planned to be changed.